tools/TestTableWriter/UiaTests.csv

```diff
@@ -62,195 +62,280 @@ FALSE,3,TextUnit_Line,5,midHistory,midHistoryP1C,5,midHistoryP5L,midHistoryP6L,F
 TRUE,1,TextUnit_Document,-5,origin,origin,0,origin,origin,FALSE
 TRUE,1,TextUnit_Document,-1,origin,origin,0,origin,origin,FALSE
 TRUE,1,TextUnit_Document,0,origin,origin,0,origin,origin,FALSE
-TRUE,1,TextUnit_Document,1,origin,origin,1,endExclusive,endExclusive,FALSE
-TRUE,1,TextUnit_Document,5,origin,origin,1,endExclusive,endExclusive,FALSE
-FALSE,1,TextUnit_Document,-5,origin,originP1C,0,origin,endExclusive,FALSE
-FALSE,1,TextUnit_Document,-1,origin,originP1C,0,origin,endExclusive,FALSE
-FALSE,1,TextUnit_Document,0,origin,originP1C,0,origin,endExclusive,FALSE
-FALSE,1,TextUnit_Document,1,origin,originP1C,0,origin,endExclusive,FALSE
-FALSE,1,TextUnit_Document,5,origin,originP1C,0,origin,endExclusive,FALSE
+TRUE,1,TextUnit_Document,1,origin,origin,1,docEnd,docEnd,FALSE

```

Comment:
This is super cool. Not that I want you to change this because your way looks more straightforward... but have you ever explored TAEF's data-table driven testing?

Reply:
This actually is secretly data table driven!